src/renderer.ts

```diff
@@ -595,6 +595,11 @@ const render = (format: Format, excludeStyles = false) =>
             const figcaption = h(FigCaption, { format, caption, credit: none });
             return styledH('figure', figureAttributes, [ styledH('div', attributes), figcaption ]);
         }
+
+        case ElementKind.AudioAtom: {
+            const { pillar } = format;
+            return h('div', { ...element, pillar: pillar.valueOf(), className: 'js-audio-atom audio-atom' }, h(AudioAtom, { ...element, pillar }))
```

Comment:
If `...element` and `pillar` exist to add attributes to the `<div>` could we use [`data-*`](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/data-*) attributes instead?

What do `valueOf` and the `audio-atom` class do here?

Reply:
Passing `Pillar.Sport` doesn't seem to work as no pillar styles are added (The screenshot above is missing the blue sport colours).

`.valueOf` isn't working either actually, I need to take another look at this.

Reply:
Had to downgrade our src-foundations library to match dotcom in order to see the pillar colours`"@guardian/src-foundations": "^1.5.0"`

cc @shtukas

Reply:
Upgraded to atoms-rendering 2.0.0 which fixes the problem